src/code/z_face_reaction.c

```diff
@@ -1,188 +1,188 @@
 #include "global.h"
 
-u16 sReactionTextIds[FACE_REACTION_SET_MAX][PLAYER_MASK_MAX] = {
-    // FACE_REACTION_SET_CARPENTER_BOSS
+u16 sReactionTextIds[MASK_REACTION_SET_MAX][PLAYER_MASK_MAX] = {
+    // MASK_REACTION_SET_CARPENTER_BOSS
     { 0x0000, 0x7124, 0x7127, 0x7126, 0x7125, 0x7127, 0x7124, 0x7125, 0x7127 },
```

Comment:
might as well just say `MaskReactionTextIds` for the name? or to better mirrot the enum name, `MaskReactionSets`? not sure which is better

Reply:
It's only used once, so just went with the full `sMaskReactionSetTextIds`